style.css

```diff
@@ -25,22 +25,48 @@ body {
   flex-wrap: wrap;
 }
 
-.header-container > a {
+.mobile-header-container > a {
   font-family: Poppins, sans-serif;
   font-weight: 600;
   text-decoration-line: none;
   color: black;
 }
 
-.header-container > nav {
+.mobile-header-container > nav {
   padding-top: 8px;
 }
 
+.desktop-header-container {
+  display: none;
+}
+
+.desktop-header-container nav {
+  margin: auto;
+}
+
+.nav-list {
+  width: 255px;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.nav-btn {
+  position: static;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
```

Comment:
- [ ] Good job with adding links to your navigation as the last review suggested but there one thing left. You you forgot to add colors to links. Usually `<a>` elements colors are not inherited from parent element (`.nav-btn` is here). I suggest adding some style to `.nav-btn a` after `.nav-btn` to make it work like this.
```css
.nav-btn a {
 color: #344563;
}
```


### Your design
![image](https://user-images.githubusercontent.com/69011963/144701206-30642d4d-923a-4e72-bfa2-92d439d49dfa.png)

### You can see figma design here
![image](https://user-images.githubusercontent.com/69011963/144701258-a2c7b4a5-962f-4b1e-98fd-d9c9a8edff37.png)

Reply:
thanks, it's done.